gacl/admin/object_search.php

```diff
@@ -42,7 +42,7 @@ function array_walk_trim(&$array_field) {
 		$query = '
 			SELECT	section_value,value,name
 			FROM	'. $gacl_api->_db_table_prefix . $object_type .'
-			WHERE	section_value='. $db->qstr($_GET['section_value']) .'
+			WHERE	section_value='. $db->qstr(applyIVPattern($_GET['section_value'])) .'
```

Comment:
Aren't html escaping and sql escaping different mechanism?
In this case, I think the qstr function is already doing the sql escaping.